packages/react/src/components/UIShell/next/UIShell.stories.js

```diff
@@ -225,7 +231,7 @@ export const HeaderBaseWActions = () => (
         aria-label="App Switcher"
         onClick={action('app-switcher click')}
         tooltipAlignment="end">
-        <AppSwitcher size={20} />
+        {/* <AppSwitcher size={20} /> */}
```

Comment:
@joshblack Using the `AppSwitcher` makes these stories blow up. Gives the "maybe you mixed up default vs named exports" error. 

Did it get renamed? I can't find any reference to it. It was updated to use `size` in https://github.com/carbon-design-system/carbon/pull/11047 but beyond that I have no clue 🤔

Reply:
@tay1orjones yeah, it was one of the deprecated icons that was removed. I think the replacement is `switcher`

Reply:
@abbeyhrt @joshblack Updated!